Makefile

```diff
@@ -15,7 +15,7 @@ endif
 LD_FLAGS = -X github.com/tendermint/tendermint/version.TMVersion=$(VERSION)
 BUILD_FLAGS = -mod=readonly -ldflags "$(LD_FLAGS)"
 HTTPS_GIT := https://github.com/tendermint/tendermint.git
-DOCKER_BUF := docker run -v $(shell pwd):/workspace --workdir /workspace bufbuild/buf
+DOCKER_PROTO_BUILDER := docker run -v $(shell pwd):/workspace --workdir /workspace tendermintdev/docker-build-proto
```

Comment:
Separately from this PR, I think we should do some combination of (a) find a new home for these build images, one where we actually have administrative credentials, and (b) define a better way to build and update this image.

Reply:
IMO we should probably just build it out of this Makefile, honestly.

Reply:
Created: https://github.com/tendermint/tendermint/issues/7272